doc/Variables.md

```diff
@@ -47,8 +47,6 @@ places where most of the dimensions are unused. However, including too few physi
 also leads to inefficient and cumbersome usage when even the simplest of observation models involve many variables.
 This is one of those "Goldilocks principle" situations.
 
-![Goldilocks principle](http://home.netcom.com/~swansont_2/goldilocks.jpg)
-
```

Comment:
This link is dead, but if there is another image that works well as an example here, feel free to add or request me to add it.